extensions/wikia/CuratedContent/CuratedContentHelper.class.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * CuratedContent Helper
+ */
+class CuratedContentHelper {
+	const STR_ARTICLE = 'article';
+	const STR_BLOG = 'blog';
+	const STR_FILE = 'file';
+	const STR_CATEGORY = 'category';
+	const STR_EMPTY_CATEGORY = 'emptyCategory';
+
+	public function processLogic( $sections ) {
```

Comment:
I'm not sure about this method name. It's not clear what it does - processing some logic isn't really self-explanatory.